workers/docker-worker/index.md

```diff
@@ -375,3 +375,47 @@ Example:
   }
 }
 ```
+
+## Coalescing
+
+In some casse, the demand for tasks outstrips the available resources to run those tasks, and pending counts on queues start to grow.
+In many cases, executing some tasks in the queue can render others unnecessary.
+For example, if the test tasks for a later version-control revision succeed, then tests for earlier revisions can be skipped.
+Such earlier tasks are "superseded" by the later task.
+
+Docker-worker supports "coalescing" multiple tasks, executing only one (the "primary task").
+It does so using a "coalescing key", determined as described in the payload schema documentation.
+
+Example:
+
+```js
+{
+    "routes": [
+        "coalesce.foo.v1.build.master"
+    ],
+    "coalescer": {
+        "routePrefix": "coalesce.foo.v1",
```

Comment:
I'm curious what is this good for?

I assume the route `"coalesce.foo.v1.build.master"` is added so that the coaleser can find the task... And I assume the coalescer knows its own prefix: `coalesce.foo.v1`...

Wouldn't it be more generic to just embed this in the URL. So the worker has not know less magic. ie:
` "url": "https://foo-coalescer.herokuapp.com/v1/superseding/build.master"`.

After all this URL is opaque to the worker... As is the entire concept of coalescence... Well, ideally it should be as much as possible.